worlds/minecraft/test/TestOptions.py

```diff
@@ -0,0 +1,54 @@
+from . import MCTestBase
+from ..Options import minecraft_options
+
+from BaseClasses import ItemClassification
+
+class AdvancementTestBase(MCTestBase):
+    options = {
+        "advancement_goal": minecraft_options["advancement_goal"].range_end
+    }
+
+    def test_beatable(self):
```

Comment:
This test is run generically for all bases now (per #1476) so this test is redundant. Just declaring the Base should run that test in particular.

Reply:
actually it doesn't unless i missed something, but that should maybe be added to the base?

Reply:
These are subclassed from worldtestbase, have nondefault options, and don't modify the setup so it should?

Reply:
ok i was mistaken and those actually weren't in the new tests lmao. added it in #1495